### PR TITLE
Add support for ratelimit-server

### PR DESCRIPTION
**Pull request by mwiencek**
Allows the server to communicate with ratelimit-server. Ported from musicbrainz-server's code & tested against a local instance of ratelimit-server (but since this stuff is fairly new to me, a thorough sanity check would be nice).

### CODE REVIEW

File: coverart_redirect/server.py [hunk 1]
```diff
@@ -59,9 +60,14 @@ class Server(object):
     def __init__(self, config):
         self.config = config
         self.engine = sqlalchemy.create_engine(self.config.database.create_url())
+        self.rate_limiter = RateLimiter(config)
 
     @Request.application
     def __call__(self, request):
+        rate_limiter_response = self.rate_limiter.check_rate_limit('frontend ip=%s' % request.remote_addr);
```
Comment (mwiencek) [inline coverart_redirect/server.py]: I'm guessing this should be changed to something new including the UA, e.g. `caa ua=`?

**Comment by mwiencek** [comment]
@mayhem I'll work on making the appropriate changes to ratelimit-server, then deploy this to coverartarchive.org/beta/ for testing.

Any suggestions about what the per-user rate-limit should be?

**Comment by mayhem** [comment]
These requests are fairly light-weight compared to other WS requests on our sites. Maybe 2/second? 3?

**Comment by gentlecat** [comment]
Is this still relevant?

**Comment by mayhem** [comment]
No, we can close this.

**Pull request closed without merging**